.dependabot/config.yml

```diff
@@ -83,6 +83,15 @@ update_configs:
           dependency_name: "org.scala-lang:scala-compiler"
       - match:
           dependency_name: "net.alchim31.maven:scala-maven-plugin"
+      # Kotlin
```

Comment:
I don't think we should add thing configuration since there are too many places where the version is being used and I don't think the bot can catch them all

Reply:
I have tried to be exhaustive when adding the configuration in the hope of overcoming that situation.

Reply:
Basically there are test projects that also use this version which explains why it's all over the place. If we find a way to consolidate them, then I think the configuration makes sense. Otherwise I would prefer we do manual updates in order to not end up with mixed versions

Reply:
The bot won't be able to update all the instances will it? If not, lets get rid of this configuration

Reply:
Oh yeah, just saw that we have a the version in a Kotlin IT template project. Not sure the bot will see and update that. Let's keep this manual for now. Thanks @geoand